rq/cli/cli.py

```diff
@@ -303,3 +307,53 @@ def resume(cli_config, **options):
     """Resumes processing of queues, that were suspended with `rq suspend`"""
     connection_resume(cli_config.connection)
     click.echo("Resuming workers.")
+
+
+@main.command()
+@click.option('--queue', '-q', help='The name of the queue.', default='default')
+@click.option('--timeout',
+              help='Specifies the maximum runtime of the job before it is interrupted and marked as failed.')
+@click.option('--result-ttl', help='Specifies how long successful jobs and their results are kept.')
+@click.option('--ttl', help='Specifies the maximum queued time of the job before it is discarded.')
+@click.option('--failure-ttl', help='Specifies how long failed jobs are kept.')
+@click.option('--description', help='Additional description of the job')
+@click.option('--depends-on', help='Specifies another job id that must complete before this job will be queued.',
+              multiple=True)
+@click.option('--job-id', help='The id of this job')
+@click.option('--at-front', is_flag=True, help='Will place the job at the front of the queue, instead of the end')
+@click.option('--retry-max', help='Maximum amound of retries', default=0, type=int)
+@click.option('--retry-interval', help='Interval between retries in seconds', multiple=True, type=int, default=[0])
+@click.option('--schedule-in', help='Delay until the function is enqueued (e.g. 10s, 5m, 2d).')
+@click.option('--schedule-at', help='Schedule job to be enqueued at a certain time formatted in ISO 8601 without '
+              'timezone (e.g. 2021-05-27T21:45:00).')
+@click.option('--quiet', is_flag=True, help='Only logs errors.')
+@click.argument('function')
+@click.argument('arguments', nargs=-1)
+@pass_cli_config
+def enqueue(cli_config, queue, timeout, result_ttl, ttl, failure_ttl, description, depends_on, job_id, at_front,
+            retry_max, retry_interval, schedule_in, schedule_at, quiet, function, arguments, **options):
+    """Enqueues a job from the command line"""
+    args, kwargs = parse_function_args(arguments)
+
+    function_string = generate_function_string(function, args, kwargs)
+    description = description or function_string
+
+    retry = None
+    if retry_max > 0:
+        retry = Retry(retry_max, retry_interval)
+
+    schedule = parse_schedule(schedule_in, schedule_at)
+
+    with Connection(cli_config.connection):
+        queue = cli_config.queue_class(queue)
+
+        if schedule is None:
+            job = queue.enqueue_call(job_func, (function, args, kwargs), {}, timeout, result_ttl, ttl, failure_ttl,
```

Comment:
I'm not sure why we're enqueueing `job_func`, we should enqueue `function` that's passed in by the user instead.

Reply:
If you look at `job_func`, you see that it allows you to execute a python file

Reply:
Then we should build this into `enqueue_call` proper. I think it's better if we merge this in in a separate PR.

Reply:
I don't think enqueueing an executable python file from python does make much sence so I implemented this only for the cli.

@selwin Please write if I should remove this from this PR.

Reply:
Sorry for the late reply. Yes, I think better to remove the ability to enqueue an executable Python file. Thanks.